core/src/main/java/edu/wpi/grip/core/operations/network/Vector2D.java

```diff
@@ -11,6 +11,7 @@
  * {@link Size}s are converted into this.
  */
 @Immutable
+@PublishableProxy({Point.class, Size.class})
```

Comment:
Do you want to use the constructor parameter to get the type?
Any reason why the type should be specified in the parameter and in the annotation?